### ISSUE TITLE
GroupedList: ExpandCollapse (chevron icon ) button should have aria-expanded to announce its state

### ISSUE THREAD

**Issue by arunarokk** [OfficeDev/office-ui-fabric-react#5751]
### Bug Report 
 
- __Package version(s)__: (6.38.0)
- __Browser and OS versions__: (Edge, Win 10)
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (No)
 
Requested priority: (High) 
 
Products/sites affected: (if applicable)
 
#### Describe the issue:
 
1. When the group by expanded or collapsed, screen reader should be able to announce its state to know the button is expanded or collapsed. 
2. In the Narrator scan mode, ExpandCollapse (chevron icon ) button is not accessible. ( the icon has role presentation & aria-hidden attributes to it, if there way to give a aria-label to the icon, then it will be accessible in scan mode)
 
#### Actual behavior:
 
ExpandCollapse (chevron icon ) button , not announcing its state. 

https://developer.microsoft.com/en-us/fabric#/components/groupedlist

**Comment by natalieethell** [comment]
Thanks for submitting this issue @arunarokk! It seems that this change is going to be a little more involved than a simple bug fix, perhaps requiring a refactor of GroupedList. Elements with roles `list` and `listitem`s are not expected to be interactive and should not have `aria-expanded` and neither should elements with roles `presentation`. 

We might want something with more of a DOM structure like the one in this W3C example: https://www.w3.org/TR/2018/WD-wai-aria-practices-1.2-20180719/examples/treeview/treeview-2/treeview-2a.html.

I will add this to the backlog and start a discussion with the team about what approach we should take to support `aria-expanded`.
